### PR TITLE
Fix README.md code sample errors

### PR DESCRIPTION
**Pull request by vinceatbluelabs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,28 +122,20 @@ you can use this:
...
                                db_engine=db_engine)
 results = records.move(source, target)
-
-df = DataFrame.from_dict([{'a': 1}]) # or make your own!
-
-source = sources.dataframe(df=df)
-target = targets.table(schema_name='myschema',
-                       table_name='mytable',
-                       db_engine=db_engine)
-results = move(source, target)
```
Comment (vinceatbluelabs) [inline README.md:146]: This looks like just a copy and past-o where the same code got put in twice!

File: README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ import os
 sqlalchemy_url = f"postgresql+psycopg2://username:{os.environ['DB_PASSWORD']}@hostname/database_name"
 db_engine = sqlalchemy.create_engine(sqlalchemy_url)
 
-df = DataFrame.from_dict([{'a': 1}]) # or make your own!
+df = DataFrame.from_dict([{'a': 1}])  # or make your own!
```
Comment (vinceatbluelabs) [inline README.md:91]: While we're at it, let's make our sample code pass flake8.

File: README.md [hunk 3]
```diff
@@ -122,28 +122,20 @@ you can use this:
 # Pull in the records-mover library - be sure to run the pip install above first!
 from records_mover import Session
 from pandas import DataFrame
-import sqlalchemy
 
 session = Session()
+session.set_stream_logging()
```
Comment (vinceatbluelabs) [inline README.md:127]: As-is, this sample code didn't log.  Let's give people a good starting point that they can debug from!

**Review by cwegrzyn — APPROVED** [review]
Hard to argue with this! :shipit:

**Pull request merged**